<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iChartProduction</class>
 <widget class="QDialog" name="iChartProduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="chkStopNextCycle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop on next cycle</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tbTerminalOut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminal Output</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tbLastRunLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOpenLog">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Log</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdCycleTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Process Run Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdStartProd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Production</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
